src/main/target/SPEEDYBEEF4/config.c

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
This file is not needed. You should remove it and also remove `#define USE_TARGET_CONFIG` from the target.h file